### ISSUE TITLE
[Bug]Complete the Title Here

### ISSUE THREAD

**Issue by xlnn** [windingwind/zotero-pdf-translate#363]
**Before Issue/提问之前**
1. See #6 and search the [Issues Page](https://github.com/windingwind/zotero-pdf-translate/issues) to make sure this is not a duplicated issue;  
2. If this is a bug with Translate, i.e. no result or error, make sure you can access the translate service and the secret is currect;  

**Describe the bug**
 I can't translate, for some reason, it's an indirect problem, sometimes it works, sometimes it doesn't.As follows!

**To Reproduce**
translate

**Expected behavior**
It is indirect problem!

**Screenshots**
![image](https://user-images.githubusercontent.com/57608826/218236327-1d45e44f-a7ea-4417-85f3-9aae19f3be53.png)

**Environment**
 - OS: Windows
 - Zotero Version(6.0.20)
 - Addon Version(1.0.8)

**Additional context**
NO!

**Comment by windingwind** [comment]
Please check if the plugin is the latest version. 

In the latest version, if the selection is not changed, the popup will be hidden.

**Comment by xlnn** [comment]
> Please check if the plugin is the latest version.
> 
> In the latest version, if the selection is not changed, the popup will be hidden.

Yes.I open the popup.As follows.This is problem is'not appearing!Thank you!

**Comment by xlnn** [comment]
When you meet this problem,first, you can translate a word,second,you can translate  sentence at normal.Thank you!
